### PR TITLE
Log warning message when trying to update LB

### PR DESCRIPTION
**Pull request by isaacd9**
- Add warning message when user attempts to modify the LB configuration of
an existing service

### CODE REVIEW

**Comment by isaacd9** [comment]
Follow up to discussion in #21. Didn't see an issue for this yet. :)

**Comment by yinshiua** [comment]
Thanks :) I will take a look at it next week!

File: ecs-cli/modules/cli/compose/entity/service/service.go [hunk 1]
```diff
@@ -215,6 +215,13 @@ func (s *Service) Up() error {
 		return s.Start()
 	}
 
+	ecsServiceName := aws.StringValue(ecsService.ServiceName)
+	if s.loadBalancer != nil {
+		log.WithFields(log.Fields{
+			"serviceName": ecsServiceName,
+		}).Warn("Unable to update load balancer on existing service.")
```
Comment (nrdlngr) [inline ecs-cli/modules/cli/compose/entity/service/service.go]: You cannot update the load balancer configuration on an existing service.

**Comment by yinshiua** [comment]
Are you okay with contributing to Apache 2.0 license?

**Comment by isaacd9** [comment]
Yes!

**Comment by yinshiua** [comment]
Thanks for your contribution :)

**Pull request merged**